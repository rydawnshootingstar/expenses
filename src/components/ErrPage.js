import React from 'react';

const ErrPage = () => (
    <div>
        404 not found dawg
        <br />
    </div>
);

export default ErrPage;