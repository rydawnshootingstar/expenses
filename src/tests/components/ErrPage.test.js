import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import ErrPage from '../../components/ErrPage';

test('should render an error page',()=> {
    const wrapper = shallow(<ErrPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});