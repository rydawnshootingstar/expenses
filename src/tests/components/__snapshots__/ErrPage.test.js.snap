// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an error page 1`] = `
<div>
  404 not found dawg
  <br />
</div>
`;
